docs/development.md

```diff
@@ -344,3 +344,145 @@ respectively.
+  }
+}
+```
+
+In addition to the provenance (in `Attestation`), the response contains
+additional fields, including the `Body.IntotoObj` object. The `payloadHash` in
+this object is the SHA256 hash of the provenance.
+
+Alternatively, the provenance can be retrieved, as part of a Rekor LogEntry,
+using the Rekor HTTP API:
+
+```bash
+$ curl --output signed-provenance-entry https://rekor.sigstore.dev/api/v1/log/entries/24296fb24b8ad77a872690e11780e927d9eddd1bc3a598e5490ad1c75fe039289a5dccc5b4d71576
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100 17907    0 17907    0     0  21765      0 --:--:-- --:--:-- --:--:-- 21758
+```
+
+The downloaded file is a JSON document. The provenance is included in the
+`attestation.data` field as a base64-encoded string, and can be extracted using
+the following command:
+
+```bash
+$ jq .[].attestation.data signed-provenance-entry | tr --delete '"' | base64 --decode
+{"_type":"https://in-toto.io/Statement/v0.1","predicateType":"https://slsa.dev/provenance/v0.2","subject":[{"name":"oak_functions_freestanding_bin","digest":{"sha256":"d059c38cea82047ad316a1c6c6fbd13ecf7a0abdcc375463920bd25bf5c142cc"}}],"predicate":{"builder":{"id":"https://github.com/slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@refs/tags/v1.2.0"},"buildType":"https://github.com/slsa-framework/slsa-github-generator@v1","invocation":{"configSource":{"uri":"git+https://github.com/project-oak/oak@refs/heads/main","digest":{"sha1":"1b128fb2556e4bdcc4f92552654bfbca9d2fb8c6"},"entryPoint":".github/workflows/provenance.yaml"},"parameters":{},"environment":{ "/* GitHub context */"}},"metadata":{"buildInvocationID":"3230206088-1","completeness":{"parameters":true,"environment":false,"materials":false},"reproducible":false},"materials":[{"uri":"git+https://github.com/project-oak/oak@refs/heads/main","digest":{"sha1":"1b128fb2556e4bdcc4f92552654bfbca9d2fb8c6"}}]}}
+```
+
+#### Downloading provenance from GitHub
+
+The provenance and the signature over it can be downloaded directly from GitHub.
+For `oak_functions_freestanding_bin` it is stored under the name
+`oak_functions_freestanding_bin.intoto.jsonl`, as an attachment on the GitHub
```

Comment:
It might be useful to provide a bit more info, such as how to find the specific workflow action intand and its related attachment, or at least a link to the specific workflow action run for the specific commit that this example uses.

Reply:
Thanks. Done.